.eslintrc

```diff
@@ -23,7 +23,7 @@
         "indent": [2, 4],
         "linebreak-style": [2, "unix"],
         "prefer-arrow-callback": 2,
-        "quotes": [2, "single"],
+        "quotes": [2, "single", "avoid-escape"],
```

Comment:
Might not be necessary since I just changed the template string to single quotes.